plugins/cpp_reparse/service/src/cppreparseservice.cpp

```diff
@@ -92,6 +92,58 @@ void CppReparseServiceHandler::getAsHTML(
   return_ = htmlFactory.str();
 }
 
+void CppReparseServiceHandler::getAsHTMLForNode(
+  std::string& return_,
+  const core::AstNodeId& nodeId_)
+{
+  if (!isEnabled())
+  {
+    return_ = "Reparse capabilities has been disabled at server start.";
+    return;
+  }
+
+  model::CppAstNodePtr astNode;
+  _transaction([&, this](){
+    astNode = _db->query_one<model::CppAstNode>(
+      AstQuery::id == std::stoull(nodeId_));
+    astNode->location.file.load();
+  });
+
+  if (!astNode)
+  {
+    LOG(error) << "No location file found for AST node #" << nodeId_;
```

Comment:
This should be a debug or warning level log. Error is given if it makes CodeCompass stop running.

Reply:
Shouldn't `CRITICAL` be used for that, though? I'll change this to a warning nonetheless.

Reply:
We don't have critical level.